### PR TITLE
VIH-7476 Logging Debug changed to Logging Information

### PR DESCRIPTION
**Pull request by ghost**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7476

### Change description ###
Swapping calls order and updating logging.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by shaed-parkar — CHANGES_REQUESTED** [review]
No need to change the logging level for each type. We can pick this story up again together, but this is actually missing the fix for the original issue in the ticket

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs [hunk 1]
```diff
@@ -306,13 +306,13 @@ private async Task SendJudgeEmailIfNeeded(HearingDetailsResponse updatedHearing,
         {
             if (newParticipantList.Any())
             {
-                _logger.LogDebug("Sending email notification to the participants");
+                _logger.LogInformation("Sending email notification to the participants");
```
Comment (RuwanikaWeerakkody) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:309]: Log levels in PROD may not log LogInformation. Please check the log levels in PROD first.

**Comment by shaed-parkar** [comment]
Also please move the send email before the group assignment as discussed

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs (hunk 1 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:310]: AppInsights will capture info and higher :)

**Review by shaed-parkar — APPROVED** [review]

**Review by RuwanikaWeerakkody — APPROVED** [review]

**Pull request merged**